README.md

```diff
@@ -203,16 +203,32 @@ my_other_config_val = 0.3
 
 ## 📓 API
 
-Each `llm` component is defined by two main settings:
+`spacy-llm` exposes a `llm` factory that accepts the following configuration options:
+
+| Argument  | Type                                        | Description                                                                         |
+| --------- | ------------------------------------------- | ----------------------------------------------------------------------------------- |
+| `name`    | `str`                                       | The name of the component.                                                          |
```

Comment:
Ah, `name` shouldn't be one of the config settings. Let me check this in the refactoring...

Reply:
It's fine, just `nlp` and `name` aren't in the config as config settings.

```suggestion
```

Reply:
Thanks !